src/providers/realtime-provider.js

```diff
@@ -294,7 +295,10 @@ export default class RealtimeProvider {
                         const datum = eventToTelemetryDatum(message.data);
                         subscriptionDetails.callback(datum);
                     }
-                } else if (this.isMdbChangesMessage(message)) {
+                } else if (this.isMdbChanges(message)) {
+                    if (!this.isParameterType(message)) {
+                        return;
+                    }
```

Comment:
Should this be instead:
```suggestion
                } else if (this.isMdbChanges(message) && !this.isParameterType(message)) {
```

?

Reply:
I don't think so. We don't want it to execute `subscriptionDetails.callback(message.data);` if it is an mdbChange but is not a parameter.

Reply:
Good catch. I meant this:

```js
 } else if (this.isMdbChanges(message) && this.isParameterType(message)) {
 ```
 
 easier to read

Reply:
That's still not quite the same.

If we have mdb changes we want to only  process it if it's a parameter type. Otherwise we want to ignore the message completely.

If I make the change you're requesting it would fall into the last `else` and call the subscription callback.